eng/Packages.Data.props

```diff
@@ -168,9 +168,8 @@
 
   <ItemGroup Condition="$(MSBuildProjectName.StartsWith('Azure.Monitor.OpenTelemetry'))">
     <!-- OpenTelemetry dependency approved for Azure.Monitor.OpenTelemetry.Exporter package only -->
-    <PackageReference Update="OpenTelemetry" Version="1.8.1" />
-    <PackageReference Update="OpenTelemetry.Exporter.InMemory" Version="1.8.1" />
-    <PackageReference Update="OpenTelemetry.Extensions.Hosting" Version="1.8.1" />
+    <PackageReference Update="OpenTelemetry" Version="1.9.0-rc.1" />
```

Comment:
QQ - is it possible that GA libraries would get a dependency on a non-GA package with this change?

Reply:
First, this only applies to my Azure.Monitor packages:
https://github.com/Azure/azure-sdk-for-net/blob/759632a80000e7149a81b5a6a26ec4656de405d6/eng/Packages.Data.props#L169-L179

Second, I would expect that if I tried to release a GA version, that the API Review tool would block due to the pre-release packages. So, to answer your question, no. :)
But this assumes we trust the API Review to catch this. :D
If this is unacceptable, I'm open to discussing alternatives.

Reply:
I had missed that it's scoped to a single library.  Thanks for the clarification!

Reply:
FYI, This PR is onboarding to 1.9.0-rc.1 which we want to test some changes.
OpenTelememetry is planning on releasing 1.9.0 GA either Friday (6/14) or Monday (6/17).

For this specific change, I don't think there's any risk of releasing any version with this RC.1 dependency.

However, in the future if having pre-release versions in the Packages.Data.props is going to raise eyebrows, please let me know! You're actually the second person to ask this exact question :)